docs/use-cases/navigate.md

```diff
@@ -1,8 +1,54 @@
 ---
-title: "Teleoperate and navigate rovers"
+title: "Teleoperate and navigate with rovers"
 linkTitle: "Teleoperate and navigate with rovers"
-weight: 60
+weight: 50
 type: "docs"
-layout: "empty"
-canonical: "tutorials/services/navigate-with-rover-base/"
+tags: ["navigation", "motion", "services", "base", "base remote control"]
+no_list: true
+description: "Use Viam to teleoperate and navigate rover bases."
+image: "/services/icons/navigation.svg"
+imageAlt: "Navigation"
+images: ["/services/icons/navigation.svg"]
 ---
+
+If you have a rover base, you can teleoperate and navigate it with Viam.
+Remotely control your base after configuring your machine on the app's **Control** tab, and set up autonomous navigation with the [navigation service](/mobility/navigation/).
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/use-cases/base-control.png" class="fill alignright" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base control card">}}
+      <b>1. Teleoperate</b>
```

Comment:
```suggestion
      <b>1. Teleoperate</b><br><br>
```
[opt]

Reply:
If we change this, please make a follow up ticket to ensure all  use case pages are consistent! Thanks!

Reply:
Think it looks ok as is

Reply:
Okay--right now two of the live use cases use br br and two do not so we may want to pick one whichever way we go. Or we don't have to be consistent since they all look slightly different anyway, with some all left aligned and some alternating, and different sizes of images! I don't care strongly just thought it might look nice.

Reply:
Make a ticket for now and we can address it separately

Reply:
Ticket made!